org.civicrm.hremergency/CRM/Hremergency/Upgrader/Steps/1004.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+trait CRM_Hremergency_Upgrader_Steps_1004 {
+
+  /**
+   * Upgrade CustomGroup, setting Emergency_Contacts is_reserved to Yes
+   * title was included here because of an issue with webform_civicrm
```

Comment:
Maybe capitalize "title" and put a full-stop after the previous line since it's a new sentence.